src/utils/parser.py

```diff
@@ -85,6 +85,14 @@
     "kernelBusyCycles": "ROUND(AVG((((End_Timestamp - Start_Timestamp) / 1000) * $max_sclk)), 0)",
 }
 
+# Section based configuration for profiling and anlysis
+section_config = {
+    "SOL": {"id": 200, "source": "0200_system-speed-of-light.yaml"},
+    "MEMCHART": {"id": 300, "source": "0300_mem_chart.yaml"},
+    "WAVEFRONT": {"id": 700, "source": "0700_wavefront-launch.yaml"},
```

Comment:
Should we rename this "LAUNCH_STATS" to better describe what we are going to show in this section?